modules/gltf/gltf_document.cpp

```diff
@@ -2406,14 +2412,15 @@ Error GLTFDocument::_serialize_meshes(Ref<GLTFState> state) {
 
 			//blend shapes
 			print_verbose("glTF: Mesh has targets");
-			if (import_mesh->get_blend_shape_count()) {
-				ArrayMesh::BlendShapeMode shape_mode = import_mesh->get_blend_shape_mode();
-				for (int morph_i = 0; morph_i < import_mesh->get_blend_shape_count(); morph_i++) {
-					Array array_morph = import_mesh->get_surface_blend_shape_arrays(surface_i, morph_i);
-					target_names.push_back(import_mesh->get_blend_shape_name(morph_i));
+			Ref<ArrayMesh> import_array_mesh = import_mesh;
```

Comment:
This line is wrong. ImporterMesh is not a compatible type with ArrayMesh

Reply:
Resolved. Tested exporting with the fox. https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/Fox/glTF-Binary/Fox.glb